cmd/tailscale/cli/file.go

```diff
@@ -203,7 +203,26 @@ func discoverPeerAPIBase(ctx context.Context, ipStr string) (base string, lastSe
 			return ft.PeerAPIURL, lastSeen, isOffline, nil
 		}
 	}
-	return "", time.Time{}, false, errors.New("target seems to be running an old Tailscale version")
+
+	// Try a bit harder for a nicer error message.
```

Comment:
If we refactored this out into a function it would read more nicely, because we could return early in a bunch of places and unindent. The signature would be a bit weird but still worth it I think.

Reply:
Done.